pkgs/development/python-modules/mathutils/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "mathutils";
+  version = "3.3.0";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```

https://nixos.org/manual/nixpkgs/unstable/#buildpythonpackage-parameters

even for projects that don't use pyproject.toml it will use pypa build gooks